engine/chassis.ts

```diff
@@ -215,9 +215,12 @@ function hashObject (o: object) {
 // 5, 3, 8
 export const TEST_UBO_COUNTS: number[] = [5, 3, -1];
 export const TEST_SAMPLER_COUNTS: number[] = [5, 3, -1];
+export const TEST_VARY_SLOT_LIM = 8;
```

Comment:
The total number of UBO slots available is a platform-specific constant, so I wouldn't recommend hard-coding this limit like this. Maybe something like `device.maxUniformBufferBindings - xxx` would be better?

Reply:
Also this constant won't be used anywhere else so the export part is redundant.